1_wqpdata/src/wqp_pull.R

```diff
@@ -80,20 +81,101 @@ partition_inventory <- function(inventory_ind, wqp_pull, wqp_state_codes, wqp_co
     dplyr::left_join(parameter_codes, by='Constituent') %>%
     dplyr::mutate(SiteType=list(site_types))
 
-  return(pull_tasks_df)
+  # write the data file and the indicator file
+  data_file <- as_data_file(ind_file)
+  saveRDS(pull_tasks_df, file=data_file)
+  gd_put(ind_file, data_file) # sc_indicate(ind_file, data_file=data_file)
+
+  invisible()
+}
+
+plan_wqp_pull_per_constituent <- function(constituents, folders, folders_item) {
+
+  partitions <- scipiper::create_task_step(
+    step_name = 'partitions',
+    target_name = function(task_name, step_name, ...) {
+      sprintf('partitions_%s', task_name)
+    },
+    command = function(task_name, ...) {
+      sprintf("readRDS('%s')",
+              file.path(folders$tmp, sprintf("partition_%s.rds", task_name)))
```

Comment:
depends on how completely this pull task remake file ends up being from the inventory task remake file, but if they're coupled, the dependency here should probably be on the .rds.ind file rather than on the .rds file itself. (and then call readRDS(sc_retrieve('partition_%s.rds.ind')), which may need to get wrapped in a function so that you can call that one unnested function from remake; my experience has been that you can't call `functionA(functionB('anything'))` as a remake `command`)